media-gfx/gimp/gimp-2.10.18.ebuild

```diff
@@ -0,0 +1,231 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
```

Comment:
no 3 yet?

Reply:
No, python3 is for 2.99/master branch (live ebuild). Branch 2.10 is still with python2